README.md

```diff
@@ -50,6 +50,7 @@ IDEs tested so far for 100% compatibility:
 - **[PHPStorm](https://github.com/dereuromark/cakephp-ide-helper/wiki/PHPStorm)** (incl. meta file generator)
 - IntelliJ
 - Atom
+- VScode
```

Comment:
Isnt the official spelling `VSCode` or `Visual Studio Code`?

Reply:
See also https://github.com/dereuromark/cakephp-ide-helper/wiki/Visual-Studio-Code that should ideally be linked then like PHPStorm above.

Reply:
yes `VS Code`
PHPStorm meta file looks does not work correctly yet in this Editor , but works if you open the meta file manually , and I can not confirm all feature of this plugin 
I started use this plugin today :)